### PR TITLE
Add LedgerClosed test (RIPD-1272)

### PR DESCRIPTION
**Pull request by mellery451**
Migrate the logic in monitor-test.js to a new jtx test
that covers the ledger_closed RPC method.

### CODE REVIEW

File: src/ripple/rpc/tests/LedgerClosed.test.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        BEAST_EXPECT(lc_result[jss::ledger_index] == 2);
+
+        env.close();
+        auto const ar_master = env.le(env.master);
+        BEAST_EXPECT(ar_master->getAccountID(sfAccount) == env.master.id());
+        BEAST_EXPECT((*ar_master)[sfBalance] == drops( 99999989999999980 ));
+
+        auto const ar_alice = env.le(alice);
+        BEAST_EXPECT(ar_alice->getAccountID(sfAccount) == alice.id());
+        BEAST_EXPECT((*ar_alice)[sfBalance] == drops( 10000000000 ));
```
Comment (miguelportilla) [inline src/ripple/rpc/tests/LedgerClosed.test.cpp]: Consider comparing against XRP instead to match how you funded.
`BEAST_EXPECT((*ar_alice)[sfBalance] == XRP(10000));`

**Comment by miguelportilla** [comment]
I left a minor comment. PR looks great. :+1:

File: src/ripple/rpc/tests/LedgerClosed.test.cpp [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        Account const alice {"alice"};
+        env.fund(XRP(10000), alice);
+
+        auto lc_result = env.rpc("ledger_closed") [jss::result];
+        BEAST_EXPECT(lc_result[jss::ledger_hash]  == "8AEDBB96643962F1D40F01E25632ABB3C56C9F04B0231EE4B18248B90173D189");
+        BEAST_EXPECT(lc_result[jss::ledger_index] == 2);
+
+        env.close();
+        auto const ar_master = env.le(env.master);
+        BEAST_EXPECT(ar_master->getAccountID(sfAccount) == env.master.id());
```
Comment (willwray) [inline src/ripple/rpc/tests/LedgerClosed.test.cpp]: This test of master account id seems redundant as it is testing jtx internals only, If I understand correctly

Comment (mellery451) [inline src/ripple/rpc/tests/LedgerClosed.test.cpp]: well, I was trying to mostly duplicate the existing monitor-root.js test, which ends up calling [this](https://github.com/ripple/rippled/blob/9b103038213bf7a6f2fb1f1d3be41977782c412e/test/testutils.js#L249). I think the whole test is fairly low value, but we are at a minimum making a ledger_entry request here.

**Comment by willwray** [comment]
👍

**Comment by nbougalis** [comment]
Merged as ad9be4dbf6185f332f2d71c4b7dafe814470d723

**Pull request closed without merging**